README.md

```diff
@@ -1,69 +1,61 @@
-# go-gh beta
+# Go library for the GitHub CLI
```

Comment:
I think about this library as being generally useful, like a GitHub SDK in Go, and not only as a companion to the CLI for writing extensions. I suppose it comes down to how you want to market it!

Reply:
Good point, but our design for go-gh will always be influenced by GitHub CLI, and I wouldn't want to market go-gh as a general-purpose Go SDK for GitHub.